api/v1alpha1/datadogmonitor_types.go

```diff
@@ -140,9 +150,9 @@ const (
 // The DatadogMonitor is triggering if one of its groups is in Alert, Warn, or No Data
 type DatadogMonitorTriggeredState struct {
 	// MonitorGroup is the name of the triggering group
-	MonitorGroup       string              `json:"monitorGroup,omitempty"`
-	State              DatadogMonitorState `json:"state,omitempty"`
-	LastTransitionTime metav1.Time         `json:"lastTransitionTime,omitempty"`
+	MonitorGroup     string              `json:"monitorGroup,omitempty"`
+	State            DatadogMonitorState `json:"state,omitempty"`
+	LastTransitionTs int64               `json:"lastTransitionTime,omitempty"`
```

Comment:
Maybe missed a prior comment, but curious why this change?

Reply:
i think i changed it to be consistent with how timestamps are reported by the API ([link](https://github.com/DataDog/datadog-api-client-go/blob/80c5e2c698b188780cb9e71debc9cd95fc2f0df3/api/v1/datadog/model_monitor_state_group.go#L18))

Reply:
Interesting, I wonder if it impacts the readability of conditions in the status as seen in `kubectl`?

Reply:
you're right, it was ugly. i switched it back